README.md

```diff
@@ -206,9 +206,9 @@ To enable this configuration use the `extends` property in your
 
 | Name                                                                                 | Description                                                                                  | 🔧  | Included in configurations                                                         |
 | ------------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------------- | --- | ---------------------------------------------------------------------------------- |
+| [`await-async-event`](./docs/rules/await-async-event.md)                             | Enforce promises from `fireEvent` methods to be handled                                      |     | ![vue-badge][] ![marko-badge][]                                                    |
```

Comment:
Have you generated this with the scripts provided in the project? They are mentioned in the PR description, so this table is automatically generated from rules meta info. The description and badges for this one doesn't seem correct.